queries/xresources/highlights.scm

```diff
@@ -0,0 +1,75 @@
+[
+  (comment)
+  (preprocessor_comment)
+] @comment @spell
+
+"#include" @keyword.import
+
+[
+  "#define"
+  "#undef"
+] @keyword.directive.define
+
+[
+  "#if"
+  "#ifdef"
+  "#ifndef"
+  "#elif"
+  "#elifdef"
+  "#elifndef"
+  "#else"
+  "#endif"
+  (directive)
+] @keyword.directive
+
+(identifier) @variable
+
+(define_directive
+  name: (identifier) @constant)
```

Comment:
Maybe `@constant` should be `@constant.macro`?

Reply:
I thought the same but the C queries use `@constant` for this and I wanted to keep them consistent with those (unless the C queries are in the wrong here).

Reply:
Oh thanks for checking that! Yeah IMO that seems wrong for the C queries to be doing that...

Reply:
I'll change the captures here and open another PR changing the captures for the C queries if that is the desired behavior.